ci/coverity_unsuccessful.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -uex
+
+rm -rf "_build.external${arch:-}"
+mkdir -p coverity
+rm -f coverity/*
+
+if [ -f "config${arch:-}.log" ]; then
+  mv "config${arch}.log" coverity/config.log-centos7-cov
```

Comment:
Should this have the distro hard-coded?  What if we wanted to switch to doing this on a different distro, or doing it on multiple distros?